src/App.tsx

```diff
@@ -1,14 +1,42 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState <Todo[]>([]);
+  const [todoId, setTodoId] = useState(0);
+  const [userId, setUserId] = useState(0);
+  const [status, setStatus] = useState('all');
+  const [query, setQuery] = useState('');
+
+  const isInQuery = (arg: string) => {
+    if (arg.toLowerCase().includes(query.toLowerCase())) {
+      return true;
+    }
+
+    return false;
+  };
```

Comment:
Method `includes` returns boolean type
```suggestion
  const isInQuery = (arg: string) => arg.toLowerCase().includes(query.toLowerCase());
```